formatTest/typeCheckedTests/expected_output/reasonComments.re

```diff
@@ -1,29 +1,39 @@
 3; /* - */
 
-3;
+3; /*-*/
```

Comment:
Ah, sounds good - surprised we don't have loc info for the atom yet. Follow up for this sounds like the best approach.